Source/Core/CylinderOutlineGeometry.js

```diff
@@ -71,8 +71,8 @@ define([
         var numberOfVerticalLines = Math.max(defaultValue(options.numberOfVerticalLines, 16), 0);
 
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(length) || length <= 0) {
-            throw new DeveloperError('options.length must be greater than 0.');
+        if (!defined(length) || length < 0) {
```

Comment:
I'll stop marking to do the same changes in the outlines, but remember that all changes must be made in both places